cmd/rewards-api/main.go

```diff
@@ -441,6 +445,37 @@ func main() {
 		if err != nil {
 			logger.Fatal().Err(err).Msg("Error occurred completing reward migrations")
 		}
+	case "ensure-attestations":
+		if len(os.Args) < 2 || os.Args[2] == "" {
+			logger.Fatal().Msg("invalid value provided for week")
+		}
+
+		week, err := strconv.Atoi(os.Args[2])
+		if err != nil {
+			logger.Fatal().Err(err).Msg("Could not parse week number.")
+		}
+
+		logger = logger.With().Int("week", week).Str("subCommand", "ensure-attestations").Logger()
```

Comment:
```suggestion
		logger = logger.With().Int("week", week).Str("subCommand", subCommand).Logger()
```
since we're switching on the subcommand we can just use that var here